src/nodes/victron-virtual.html

```diff
@@ -254,24 +269,34 @@
             <option value="grid">Grid meter</option>
             <option value="meteo">Meteo</option>
             <option value="pvinverter">PV inverter</option>
-            <option value="switch">Switch</option>
+            <option value="switch">Switch (deprecated)</option>
             <option value="tank">Tank sensor</option>
             <option value="temperature">Temperature sensor</option>
         </select>
     </div>
-    <div class="form-row input-acload" style="display:none;">
+    <div class="form-row" id="switch-deprecation-notice-row" style="display: none;">
+        <div class="deprecation-notice">
+            <i class="fa fa-info-circle fa-2x"></i>
+            <div>
+                <strong>Deprecation Notice:</strong> The "Switch" device type is deprecated in the "Virtual Device" node.
+                Please use the dedicated <strong>"Virtual Switch"</strong> node for new switch configurations.
+                This Virtual Device will continue to work for now, but we recommend migrating to the new node.
```

Comment:
"... continue to work, but ..." may make me anxious as customer.

How about "... continue to work until release 1.7. We recommend migration to the new node." or similar?